packages/eslint-config/index.js

```diff
@@ -157,6 +157,8 @@ export const config = [
           ],
         },
       ],
+      "turbo/no-undeclared-env-vars": "off",
```

Comment:
This is not needed and it should be set to `on` (default) - so we won't forget to add env var to turbo.json

Reply:
yea AI wrote it